underscore.js

```diff
@@ -1213,6 +1213,11 @@
     // Assume equality for cyclic structures. The algorithm for detecting cyclic
     // structures is adapted from ES 5.1 section 15.12.3, abstract operation `JO`.
 
+    // Compare array lengths to determine if a deep comparison is necessary.
+    if (areArrays && a.length !== b.length) {
+      return false;
+    }
+    
```

Comment:
not equal

On Mon, Jan 11, 2016 at 6:37 PM, ven <notifications@github.com> wrote:

> In underscore.js
> <https://github.com/jashkenas/underscore/pull/2406#discussion_r49396317>:
>
> > @@ -1213,6 +1213,11 @@
> >      // Assume equality for cyclic structures. The algorithm for detecting cyclic
> >      // structures is adapted from ES 5.1 section 15.12.3, abstract operation `JO`.
> >
> > +    // Compare array lengths to determine if a deep comparison is necessary.
> > +    if (areArrays && a.length !== b.length) {
> > +      return false;
> > +    }
> > +
>
> @megac <https://github.com/megac> what did you mean by "are neq" in your
> first comment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jashkenas/underscore/pull/2406/files#r49396317>.
>